<!-- -->
<!-- Convert the depth image and the rgb image into a xyzrgb Point Cloud -->
<launch>

    <!--node name="update_frame_id" pkg="depth_image_proc" type="update_frame_id.py" output="screen" /-->

    <!-- Nodelet manager for this pipeline -->
    <node pkg="nodelet" type="nodelet" args="manager" name="standalone_nodelet" output="screen"/>


    <!-- Convert to point cloud -->
    <node pkg="nodelet" type="nodelet" name="depth_image_proc" args="load depth_image_proc/point_cloud_xyzrgb standalone_nodelet --no-bond">
   
        <!--remap from="camera_info" to="/airsim_node/drone_1/front_center_custom_color/Scene/camera_info"/-->
        
        <!--remap from="depth_registered/image_rect" to="/airsim_node/drone_1/front_center_custom_depth/DepthPerspective"/ -->
        
        <!--remap from="image_rect_color" to="/airsim_node/drone_1/front_center_custom_color/Scene"/-->
        
        <!--remap from="depth_registered/points" to="/camera/depth/points" /-->

    </node>

</launch>

<!-- -->